src/DynamoCoreWpf/ViewModels/Watch3D/IWatch3DViewModel.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Windows.Input;
+using Autodesk.DesignScript.Interfaces;
+using HelixToolkit.Wpf.SharpDX;
+
+
+namespace Dynamo.Wpf.ViewModels.Watch3D
```

Comment:
I'm okay with this file being under `Watch3D` instead of `Interfaces` folder, but I guess we should really start documenting our public contracts. A role model of API documentation can be found for `IViewExtension`, which explains the purposes, the acceptable argument values, and the expected return values. I was thinking about adding a new YT task to track this but since this is new/small, we might as well just add some here and be done with it. Does that make sense?